compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -153,6 +153,10 @@ pub trait MirPass<'tcx> {
         to_profiler_name(self.name())
     }
 
+    fn min_mir_opt_level(&self) -> usize {
+        0
+    }
```

Comment:
This shouldn't have a default, with my suggested changes.

Reply:
Removing the default just means adding this to about 30 passes which are always enabled, which is a lot of duplication